wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift

```diff
@@ -122,6 +125,70 @@ final class SelfProfileViewController: UIViewController {
                 }
             )
         }
+
+        if DeveloperFlag.multibackend.isOn {
+            let accountSwitcherViewController = makeAccountSwitcherViewController(
+                settingsCellDescriptorFactory: settingsCellDescriptorFactory
+            )
+            self.bottomController = accountSwitcherViewController
+            self.accountSwitcherViewController = accountSwitcherViewController
+        } else {
+            let settingsController = rootGroup.generateViewController()! as! SettingsTableViewController
+            self.bottomController = settingsController
+            self.settingsController = settingsController
+        }
+    }
+
+    private func makeAccountSwitcherViewController(settingsCellDescriptorFactory: SettingsCellDescriptorFactory)
+        -> AccountSwitcherHostingController {
+        var options = [Option.addAccountOption(action: {
+            settingsCellDescriptorFactory
+                .addAccountOrTeamCell().select(.none, sender: UIView())
+        })]
+        if userSession.selfUser.canManageTeam == true {
+            options.append(Option.manageTeamOption(action: { [weak self] in
+                let controllerToShow = BrowserViewController(url: URL.manageTeam(source: .settings))
+                controllerToShow.modalPresentationCapturesStatusBarAppearance = true
+                self?.present(controllerToShow, animated: true, completion: .none)
+            }))
+        }
+
+        let accountManager = SessionManager.shared?.accountManager
+        let accounts = (accountManager?.accounts ?? [])
+            .filter {
+                !$0.isEqual(accountManager?.selectedAccount)
```

Comment:
question: what's the difference here with `$0 != accoundManager?.selectedAccount` ?